### ISSUE TITLE
ImportError: /lib64/libstdc++.so.6: version `GLIBCXX_3.4.20' not found (required by /home/ekofman/new_anaconda3/envs/mxfold2_python38/lib/python3.8/site-packages/mxfold2/interface.cpython-38-x86_64-linux-gnu.so)

### ISSUE THREAD

**Issue by ekofman** [keio-bioinformatics/mxfold2#6]
Looks like it installed successfully, but now when I try to run it I get the following error:

`ImportError: /lib64/libstdc++.so.6: version `GLIBCXX_3.4.20' not found (required by /home/ekofman/new_anaconda3/envs/mxfold2_python38/lib/python3.8/site-packages/mxfold2/interface.cpython-38-x86_64-linux-gnu.so)`

Anybody seen this before/ know how to resolve it? Thank you!

**Comment by ekofman** [comment]
This is the solution: https://github.com/BVLC/caffe/issues/4953

**Comment by samLeung24** [comment]
Yes I have worked for the entire afternoon trying to solve the installation problem. Thank you! 
`conda install libgcc` does work
